integration-tests/solr/pom.xml

```diff
@@ -80,4 +98,52 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <!-- force the locale as we want to explicitly test message interpolation -->
+                        <javax.net.ssl.trustStore>${solr.trust-store}</javax.net.ssl.trustStore>
```

Comment:
This will not work OotB on the platform, where the tests are run from test jars and the local resources are not available. I see that this also cannot be configured to pull the trust store from the jar, because the system property expects a local file. Hence a special step will be needed on the platform to unpack the test jar and set the trust store accordingly. Maybe we could make creating that preprocessing step easier by some changes here. Let me file a followup for that.

Reply:
@ppalaga I can't tell if the error comes from this
```
 :+1: Caused by: io.quarkus.bootstrap.resolver.maven.DeploymentInjectionException: io.quarkus.bootstrap.resolver.maven.BootstrapMavenException: Failed to read descriptor of io.quarkus:quarkus-resteasy-jackson-deployment:jar:1.10.0.Final
....
Caused by: org.eclipse.aether.transfer.ArtifactTransferException: Could not transfer artifact io.quarkus:quarkus-resteasy-jackson-deployment:pom:1.10.0.Final from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/io/quarkus/quarkus-resteasy-jackson-deployment/1.10.0.Final/quarkus-resteasy-jackson-deployment-1.10.0.Final.pom
```
or from the code. 
If those variables `javax.net.ssl.trustStore`, `javax.net.ssl.trustStorePassword` set on pom.xml are not recognized, we can have the same exception on SSL handshake. But we should have first the logs of the app starting, and 7/20 tests failing.